src/workspace-hack/Cargo.toml

```diff
@@ -66,7 +66,7 @@ hmac = { version = "0.12", default-features = false, features = ["reset"] }
 hyper = { version = "0.14", features = ["full"] }
 indexmap-dff4ba8e3ae991db = { package = "indexmap", version = "1", default-features = false, features = ["serde"] }
 indexmap-f595c2ba2a3f28df = { package = "indexmap", version = "2", features = ["serde"] }
-itertools = { version = "0.11" }
+itertools = { version = "0.10" }
```

Comment:
Why degraded?

Reply:
Check cargo.lock. It's funny

Reply:
Oh it bumped `workspace-hack` again first. 🤔